plugins/kubernetes/app/models/kuber_deploy_service.rb

```diff
@@ -46,6 +47,20 @@ def create_services!
     end
   end
 
+  def delete_replication_controllers!
+    kuber_release.release_docs.each do |release_doc|
+      log 'deleting existing ReplicationControllers', role: release_doc.kubernetes_role.name
+      labels = {project: release_group.build.project_name, role: release_doc.kubernetes_role.label_name}
```

Comment:
I read somewhere (can't find the reference now, it might have been the Pickaxe book) that it's good practise to use spaces for block one-liners, but not for hashes, in order to make the two visually distinct. It seemed reasonable, but coherence trumps convention :)